packages/module/package.json

```diff
@@ -27,32 +27,30 @@
   "homepage": "https://github.com/patternfly/react-component-groups#readme",
   "publishConfig": {
     "access": "public",
-    "tag": "prerelease"
+    "tag": "alpha"
```

Comment:
Do we need to be releasing on alpha from this channel rather than just prerelease?